scripts/process/geom_dataframe.R

```diff
@@ -4,8 +4,8 @@
 #' @details 
 #' This function takes a single required dependency, which is `spatial_data`
 #' Presently, `spatial_data` is Spatial{X}DataFrame, but this will be updated to `sf`
-#' if `eval_args` are present, they are used to convert the `data` slot by evaluating
-#' simple functions w/ variables in `data` (skipped if `eval_args` isn't present)
+#' if `attributes` are present, they are used to convert the `data` slot by evaluating
```

Comment:
not `data` but `spatial_data`, right? and maybe rather than "evaluating simple functions w/ variables" say something about the `{ }` templating syntax

Reply:
thanks - I had partially updated the docs from an old commit, and didn't update when I switched to "{var}"